src/olympia/addons/indexers.py

```diff
@@ -317,7 +318,8 @@ def extract_document(cls, obj):
         attrs = ('id', 'average_daily_users', 'bayesian_rating',
                  'contributions', 'created',
                  'default_locale', 'guid', 'hotness', 'icon_hash', 'icon_type',
-                 'is_disabled', 'is_experimental', 'last_updated',
+                 'is_disabled', 'is_experimental', 'is_recommended',
```

Comment:
If you add it there, it needs to always exist - see the `getattr` below, it doesn't have a default value, because it doesn't know what makes sense for each individual field. Since it's a new property, you should explicitly do a separate `getattr(obj, `is_recommended`, False)` or something like that instead, then later, when we've reindexed, we'll be free to change this back to your original version

Reply:
Mmm, I thought that would be a problem in the _serializer_ not the indexer?  `_attach_fields` would try to do `value = data.get(field_name, None)` and then `setattr(obj, field_name, value)` (i.e. `setattr(addon_instance, 'is_recommended', data.get('is_recommended', None))`) and end up setting it to `None`.  `None` isn't strictly `false` I suppose... and we are returning it...

Reply:
Maybe always cast it to `bool` in `to_representation`?

Reply:
Sorry, totally misread, it is indeed only a problem in the serializer, not the indexer.

Reply:
I think it's fine as is. It can only be `None` initially... it will be `False` after we reindex...